about.html

```diff
@@ -0,0 +1,190 @@
+              alt="logo image"
+            />
+          </a>
+          <button
+            class="navbar-toggler collapsed d-flex d-md-none flex-column justify-content-around"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarNav"
+            aria-controls="navbarNav"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="toggler-icon top-bar"></span>
+            <span class="toggler-icon middle-bar"></span>
+            <span class="toggler-icon bottom-bar"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+              <li class="nav-item me-5 home-menu-about">
+                <a class="nav-link" aria-current="page" href="index.html"
+                  >Home</a
+                >
+              </li>
+              <li class="nav-item me-5">
+                <a
+                  class="nav-link about-active"
+                  aria-current="page"
+                  href="index.html"
+                  >About</a
+                >
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link" href="index.html#service-section"
+                  >Services</a
+                >
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link" href="index.html#partners-section"
+                  >Sponsors</a
+                >
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link">Forum</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="banner-section banner-section-about-page">
+        <div class="background-image background-image-about-page"></div>
+        <div class="banner-text-content banner-text-content-about">
+          <h1 class="mb-3 text-center">Hello Developers!</h1>
+          <h2 class="fw-bold text-center">
+            ReviewMe, GET INSTANT HELP, <br />
+            MEET PROFESSIONALS AND <br />
+            MOVE 5x FASTER
+          </h2>
+          <div class="bg-white site-description-div">
+            <p class="p-3 text-center">
+              Find qualified code reviewers, get feedback on your code, and
+              improve the quality of your code. Learn new skills, and give
+              feedback to the software development community.
+            </p>
+          </div>
+          <div class="text-center">
+            <p class="mt-3">
+              Please Contact us per Email for any further questions about
+              ReviewMe! <br />
+              <span class="fw-bold text-decoration-underline"
+                >brunokambere@gmail.com</span
+              >
+            </p>
+          </div>
+        </div>
+      </section>
```

Comment:
- [ ] Kindly make sure to follow the design guidelines as much as possible, for this section, kindly address the following:-

    - You need to use the proper font, and font size to match the actual design.
    -  You need to add the proper paddings, spacings, and margins to adjust the layout.

| Your design | Actual design |
| ------------- | ------------- |
| ![image](https://github.com/kambereBr/ReviewMe/assets/102612396/60510659-52b4-45cd-8d3d-97add1f2f7b1) | ![image](https://github.com/kambereBr/ReviewMe/assets/102612396/ebaaeabe-98b8-4db8-ac1d-eaa1e76cfa2f)|